service/src/test/java/bio/terra/tanagra/service/VumcAdminServiceTest.java

```diff
@@ -24,12 +25,14 @@ public class VumcAdminServiceTest {
   @Autowired private VumcAdminService vumcAdminService;
 
   @Test
+  @Disabled
   void version() throws ApiException {
     SystemVersion systemVersion = vumcAdminService.version();
     LOGGER.info("version returned success: {}", systemVersion);
   }
 
   @Test
+  @Disabled
```

Comment:
Please add a comment about why these tests are disabled and how to run them locally, e.g.

`@Disabled("VUMC admin service OAuth client id is not checked into this repo. You can run this test locally by setting the vumc-admin properties in application-test.yaml.")`